### PR TITLE
Feature/revsdl 2476 unit tests

### PR DESCRIPTION
**Pull request by bonewell**
- Unit tests for all commands
- Few fixes
- Little refactoring

### CODE REVIEW

**Comment by bonewell** [comment]
@DKlimenko 
@VladSemenyuk 
@KKarlash 
Review, please.

**Comment by VladSemenyuk** [comment]
Reviewed

File: src/components/vr_module/include/vr_module/commands/activate_service.h [hunk 1]
```diff
@@ -56,7 +57,8 @@ class ActivateService : public TimedCommand, public event_engine::EventObserver<
           vr_mobile_api::RPCName>& event);
 
  private:
-  VRModule* module_;
+  bool SendResponse(vr_hmi_api::ResultCode code);
+  ServiceModule* module_;
```
Comment (KKarlash) [inline src/components/vr_module/include/vr_module/commands/activate_service.h]: please separate method and data members by empty line

File: src/components/vr_module/include/vr_module/commands/process_data.h [hunk 2]
```diff
@@ -56,7 +56,9 @@ class ProcessData : public TimedCommand, public event_engine::EventObserver<
           vr_mobile_api::RPCName>& event);
 
  private:
-  VRModule* module_;
+  bool SendResponse(vr_hmi_api::ResultCode code, const std::string* text = NULL,
+                    const std::string* info = NULL);
+  ServiceModule* module_;
```
Comment (KKarlash) [inline src/components/vr_module/include/vr_module/commands/process_data.h]: please separate method and data members by empty line

File: src/components/vr_module/src/commands/support_service.cc [hunk 3]
```diff
@@ -77,7 +77,11 @@ void SupportService::on_event(
   vr_hmi_api::SupportServiceResponse response;
   if (message.has_params() && response.ParseFromString(message.params())) {
     bool supported = response.result() == vr_hmi_api::SUCCESS;
-    module_->set_supported(supported);
+    if (supported) {
```
Comment (KKarlash) [inline src/components/vr_module/src/commands/support_service.cc]: ternary operator will be shorter, won't it?

**Comment by KKarlash** [comment]
Reviewed

File: src/components/vr_module/include/vr_module/commands/activate_service.h (hunk 1 above)
Comment (bonewell) [inline src/components/vr_module/include/vr_module/commands/activate_service.h]: I have only 2 lines. I don't see real cause to do your suggestion.

File: src/components/vr_module/src/commands/support_service.cc (hunk 3 above)
Comment (bonewell) [inline src/components/vr_module/src/commands/support_service.cc]: The shortest code is not always readable code.
Ternary operator is good to use for assign variable in my opinion.

**Comment by DKlimenko** [comment]
reviewed

**Pull request merged**